sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -29,13 +29,30 @@ export interface ServiceBusClientOptions {
    */
   dataTransformer?: DataTransformer;
   /**
-   * @property The WebSocket constructor used to create an AMQP connection
-   * over a WebSocket. In browsers, the built-in WebSocket will be  used by default. In Node, a
-   * TCP socket will be used if a WebSocket constructor is not provided.
+   * @property
+   * Options to configure the channelling of the AMQP connection over Web Sockets.
+   */
+  webSocketOptions?: WebSocketOptions;
+}
+
+/**
+ * Options to configure the channelling of the AMQP connection over Web Sockets.
+ */
+export interface WebSocketOptions {
```

Comment:
Now that we are duplicating this in Event Hubs and Service Bus, how about moving this interface to `@azure/core-amqp`?

Reply:
good idea, will do.